Documentation/Signoffs/Heartbeat Respiratory Subsystem.md

```diff
@@ -2,19 +2,19 @@
+The purpose of this subsystem is to measure the presence of a heartbeat as well as measure the respiratory rate of a person contactlessly. This subsystem will be composed of a singular radar that will perform both of these actions. The measuring of a person’s presence of a heartbeat as well as respiratory rate will be done by measuring the rise and fall rates of a person’s chest to measure the respiratory rate, and the rise and fall rates of the area around the heart to measure the presence of a heartbeat. A series of signals will be sent and received to measure these changes. The raw data will be put through an amplifier/filter and then be converted to a digital signal that is sent to a computing system to be put through a processing algorithm.
 
 ## Specs and Constraints			
 
 
 | Description | Constraint | Source of Constraint |
 |-------------|------------|----------------------|
-| Heartbeat analysis | Must sense Beats per minute from 27 to 200 bpm (or frequencies from 0.45 Hz to 3.33 Hz) | From Conceptual Design, from Heartbeat/Respiratory System section |
-| Respiratory sensor | Must sense breaths per minute from 7.8 to 30 bpm (or frequencies from 0.13 Hz to 0.50 Hz) | From Conceptual Design, from Heartbeat/Respiratory System section |
+| Heartbeat analysis | Must sense Beats per minute from 30 to 210 bpm (or frequencies from 0.5 Hz to 3.5 Hz)| From Conceptual Design, from Heartbeat/Respiratory System section[5][6]|
+| Respiratory sensor |Must sense breaths per minute from 1 to 70bpm (or frequencies from 0.017 Hz to  1.17 Hz) | From Conceptual Design, from Heartbeat/Respiratory System section analyzed from START model[7] |
 | Distance | Must detect and measure heartbeat and breath rate from at least 1 meter away | From DARPA constraints |
 | Weight | Entire system must weigh less than 6 pounds, this subsystem should weight 1.5 lbs or less | From DARPA constraints/ Conceptual design |
 | Safety | Radar must not emit at a frequency over 10 GHz with a power density of 1000 W/m^2 in order to keep the radar skin and eye safe. | From World Health Organization (WHO) |
-
+| Operational Amplifier | Radar input signal amplitude  must be increased by a gain of 6 in order to match the input range of the AD HAT board ADC module | From further analysis of the radar and computing subsystems. |
```

Comment:
This is ambiguous. Why a gain of 6? What are the input expectations voltage range, current, frequency, etc? What are the required output characteristics - voltage range, current, frequency, etc?

Reply:
This has not been addressed. What are the input expectations to the amp? What are the output expectations?